packages/gatsby/src/internal-plugins/internal-data-bridge/gatsby-node.js

```diff
@@ -157,3 +157,7 @@ emitter.on(`DELETE_PAGE`, action => {
   const node = getNode(nodeId)
   boundActionCreators.deleteNode({ node })
 })
+
+exports.createSchemaCustomization = ({ actions }) => {
+  actions.createTypes(require(`../../schema/types/built-in-types`)())
+}
```

Comment:
Not sure if `internal-data-bridge` is the right place for this. Open to suggestions.

Reply:
Moved this from `internal-data-bridge` to `schema/index.js`. This has a side effect that now those types show up in some test snapshots.